src/pseudopeople/configuration/generator.py

```diff
@@ -185,7 +184,7 @@ def get_noise_type_dict(noise_type: NoiseType, is_no_noise: bool) -> dict[str, f
 
 def add_overrides(
     noising_configuration: LayeredConfigTree,
-    overrides: Dict,
+    overrides: dict,
```

Comment:
Why isn't mypy complaining about the unparameterized dicts in this file?

Reply:
I don't think we've updated psp at all for mypy yet, right?

Reply:
I'm not sure why mypy requires typing the keys and values in some cases and ignores it in others.

Reply:
@stevebachmeier I'm not sure I understand your question - the branch this is merging into is for mypy updates in psp.